### PR TITLE
Napa compliance

### PR DESCRIPTION
**Pull request by aclum**
This PR merges in python code used during re-iding. There are no changes to the schema or example data. No migrations required, just trying to get old branches merged.

### CODE REVIEW

**Comment by aclum** [comment]
linked issues https://github.com/microbiomedata/issues/issues/430 and [Re-ID NOM and Metabolomics OmicsProcessing, DataObjects, AnalysisActivies#524](https://github.com/microbiomedata/issues/issues/524)

**Comment by aclum** [comment]
Closing since this merges to main-old, not main.

**Comment by eecavanna** [comment]
I recommend moving these scripts into a subdirectory (e.g. `nmdc_schema/re_id_script`). I think that will distinguish them from general-purpose schema-related code. Note: Moving them might require updating some `import` statements.

**Comment by aclum** [comment]
@turbomam  to suggest a different directory to put these scripts.

**Comment by aclum** [comment]
Ideally it would be something more generic, I've had to make other ad hoc scripts  (ie https://github.com/microbiomedata/nmdc-schema/blob/main/nmdc_schema/identify_workflow_duplicates_emp500.py) for data clean up and I'd like this to be co-located.

**Comment by turbomam** [comment]
@aclum I made a commit in this branch that moves the Python files and adds a few lines to the top of the README. If that's OK with you, then you can merge as far as I'm concerned.

**Comment by turbomam** [comment]
Sorry, because I did a filesystem move and edited the README, no useful diff shows up 

here's what I added:

```
This code has been executed, the results were accepted, and it should not need to be run again. It is being included as
documentation. The Python files have been moved from `nmdc_schema/` to `nmdc_schema/completed_napa_compliance/` and they
shouldn't be expected to work from that location.

One shouldn't assume that the installation notes below are intended for any other nmdc-schema development.
```

**Comment by turbomam** [comment]
oops, I missed your "more generic" comment too

**Comment by turbomam** [comment]
@aclum @eecavanna I moved the scripts once again to `src/scripts/task_specific_code/` and added a little more to the README.md

Please let me know if this is acceptable to you

We can move [identify_workflow_duplicates_emp500.py](https://github.com/microbiomedata/nmdc-schema/blob/main/nmdc_schema/identify_workflow_duplicates_emp500.py) into `src/scripts/task_specific_code/` in a separate PR. That file isn't present in this branch.

**Review by turbomam — APPROVED** [review]

File: src/scripts/task_specific_code/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+You can use [`black`](https://black.readthedocs.io/en/stable/) to format the Python code you write, by running:
+
+```shell
+python -m black /path/to/python/code.py
+```
+
+For example:
+
+```shell
+python -m black nmdc_schema/connect_napa_mongo.py
```
Comment (eecavanna) [inline src/scripts/task_specific_code/README.md:38]: This still reflects the original path

**Review by eecavanna — CHANGES_REQUESTED** [review]
I'm OK with the new folder name. I requested a change to some example commands, since they use the old folder name. 

Out sick this afternoon. Once the above is resolved, I'm OK with this being merged in without my involvement/review.

**Comment by aclum** [comment]
This is acceptable to me, I've addressed Eric's comments about the paths in the README.md file and have requested a re-review.

**Review by eecavanna — APPROVED** [review]

**Pull request merged**